src/components/Intro/Intro.js

```diff
@@ -0,0 +1,45 @@
+import styles from "./Intro.module.css";
+
+function Intro() {
+  return (
+    <section className={styles.wrapper}>
+      <title className={styles.title}>Чего мы там не видели?</title>
+      <p className={styles.introText}>
+        По опросам ВЦИОМ, 95% россиян мечтают куда-нибудь поехать, но только 36%
+        планируют провести отпуск в родной стране. Мол, чего мы тут, дома, не
+        видели? На самом деле, Россия — это целая вселенная с ласковым морем
+        юга, густыми лесами Саян и суровыми льдами плато Путорана. А ещё увидеть
+        все эти красоты можно без миллионов на счету, загранпаспорта и
+        многочасовых перелетов. Как, например, Вера Башмакова — смелая молодая
+        мама, которая взяла в охапку троих детей, усадила их в свою «Ладу» и
+        проехала 20 тысяч километров по родной стране. Мы выбрали и описали
+        некоторые интересные места, достойные вашего отпуска.
+      </p>
+      <ul className={styles.introList}>
+        <li className={styles.introItem}>
+          <span className={styles.boldItem}>Часовых поясов</span> 11
+        </li>
+        <li className={styles.introItem}>
+          <span className={styles.boldItem}>
+            Объектов природного наследия ЮНЕСКО
+          </span>{" "}
+          12
```

Comment:
`12` should not be without a tag. For example `<span>12</span>`
Same for the rest cases

Reply:
12 is an item in <li>, though. I have "Объектов природного наследия ЮНЕСКО" wrapped in <span> because it needs to be formatted and 12 does not need to be formatted, so it's basically <li>12</li>. I still need span?

Reply:
oh, github formats the tags lol. The bullets above are < li > tags.